src/modules/workspaces/workspaces.controller.ts

```diff
@@ -65,7 +76,9 @@ export class WorkspacesController {
     return this.workspaceUseCases.getWorkspaceTeams(user, workspaceId);
   }
 
-  @Get('/:workspaceId/teams/:teamId/members')
+  @Get('/teams/:teamId/members')
```

Comment:
No, I just removed the workspace because the teams have unique IDs. However, let's add it again just to be able to enforce any type of check just in case.